### ISSUE TITLE
Bug: Cant load 1.3.1/1.3.1.1/1.3.1.2 versions (i can 1.3.0)

### ISSUE THREAD

**Issue by GioGr** [cabarius/ToyBox#187]
When i try to load 1.3.1.1/2 versions 
i always get this 
```
[ToyBox] Version '1.3.1.2'. Loading.
[ToyBox] [Error] Error trying to call 'ToyBox.Main.Load'.
[ToyBox] [Exception] TargetInvocationException - Exception has been thrown by the target of an invocation.
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> HarmonyLib.HarmonyException: Patching exception in method null ---> System.ArgumentException: Undefined target method for patch method static System.Void ToyBox.BagOfPatches.SummonUnitsAfterArmyBattle_Patch::Prefix(Kingmaker.Armies.TacticalCombat.TacticalCombatResults& results)
  at HarmonyLib.PatchClassProcessor.PatchWithAttributes (System.Reflection.MethodBase& lastOriginal) [0x00047] in <88b084d277f344aa9c2904c640976875>:0 
  at HarmonyLib.PatchClassProcessor.Patch () [0x00059] in <88b084d277f344aa9c2904c640976875>:0 
   --- End of inner exception stack trace ---
  at ToyBox.Main.Load (UnityModManagerNet.UnityModManager+ModEntry modEntry) [0x0009d] in <1a55cdc230b44b44909613321ca575a0>:0 
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00032] in <eae584ce26bc40229c1b1aa476bfa589>:0 
   --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00048] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at UnityModManagerNet.UnityModManager+ModEntry.Invoke (System.String namespaceClassnameMethodname, System.Object& result, System.Object[] param, System.Type[] types) [0x00011] in <ed510dbfd2164cfaa2793f70ef8091df>:0 
[ToyBox] Not loaded.
[ToyBox] Loading time 0.41 s.
```
```
[ToyBox] Version '1.3.1.1'. Loading.
[ToyBox] [Error] Error trying to call 'ToyBox.Main.Load'.
[ToyBox] [Exception] TargetInvocationException - Exception has been thrown by the target of an invocation.
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> HarmonyLib.HarmonyException: Patching exception in method null ---> System.ArgumentException: Undefined target method for patch method static System.Void ToyBox.BagOfPatches.SummonUnitsAfterArmyBattle_Patch::Prefix(Kingmaker.Armies.TacticalCombat.TacticalCombatResults& results)
  at HarmonyLib.PatchClassProcessor.PatchWithAttributes (System.Reflection.MethodBase& lastOriginal) [0x00047] in <88b084d277f344aa9c2904c640976875>:0 
  at HarmonyLib.PatchClassProcessor.Patch () [0x00059] in <88b084d277f344aa9c2904c640976875>:0 
   --- End of inner exception stack trace ---
  at ToyBox.Main.Load (UnityModManagerNet.UnityModManager+ModEntry modEntry) [0x0009d] in <d2744263f6784e688f34b5c40c28fc08>:0 
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00032] in <eae584ce26bc40229c1b1aa476bfa589>:0 
   --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00048] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at UnityModManagerNet.UnityModManager+ModEntry.Invoke (System.String namespaceClassnameMethodname, System.Object& result, System.Object[] param, System.Type[] types) [0x00011] in <ed510dbfd2164cfaa2793f70ef8091df>:0 
[ToyBox] Not loaded.
[ToyBox] Loading time 0.45 s.
```
i load 1.2.13 fine
tried earlier and i can load 1.3.0 but not 1.3.1
```
[ToyBox] Version '1.3.1'. Loading.
[ToyBox] [Error] Error trying to call 'ToyBox.Main.Load'.
[ToyBox] [Exception] TargetInvocationException - Exception has been thrown by the target of an invocation.
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> HarmonyLib.HarmonyException: Patching exception in method null ---> System.ArgumentException: Undefined target method for patch method static System.Void ToyBox.BagOfPatches.SummonUnitsAfterArmyBattle_Patch::Prefix(Kingmaker.Armies.TacticalCombat.TacticalCombatResults& results)
  at HarmonyLib.PatchClassProcessor.PatchWithAttributes (System.Reflection.MethodBase& lastOriginal) [0x00047] in <88b084d277f344aa9c2904c640976875>:0 
  at HarmonyLib.PatchClassProcessor.Patch () [0x00059] in <88b084d277f344aa9c2904c640976875>:0 
   --- End of inner exception stack trace ---
  at ToyBox.Main.Load (UnityModManagerNet.UnityModManager+ModEntry modEntry) [0x0009d] in <d0ae9164f25148f99093d6d1dc2f75f3>:0 
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00032] in <eae584ce26bc40229c1b1aa476bfa589>:0 
   --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00048] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <eae584ce26bc40229c1b1aa476bfa589>:0 
  at UnityModManagerNet.UnityModManager+ModEntry.Invoke (System.String namespaceClassnameMethodname, System.Object& result, System.Object[] param, System.Type[] types) [0x00011] in <ed510dbfd2164cfaa2793f70ef8091df>:0 
[ToyBox] Not loaded.
[ToyBox] Loading time 0.40 s.
```

**Comment by GioGr** [comment]
how i add label aaaaaa

**Comment by cabarius** [comment]
What version of the game are you running?  It may the code being patched is only on WotR 1.02g

**Comment by cabarius** [comment]
I think for now I will disable ToyBox.BagOfPatches.SummonUnitsAfterArmyBattle_Patch for now

**Comment by ArcaneTrixter** [comment]
Looks like a method was renamed in a patch last week, and this is using the new method name. Sorry about that!

**Comment by GioGr** [comment]
> What version of the game are you running? It may the code being patched is only on WotR 1.02g

you were right was using 1.0.0 
switched to 1.0.2g and now it loads fine ty :)
